proto/p4/p4runtime.proto

```diff
@@ -314,6 +315,33 @@ message CounterData {
   int64 packet_count = 2;
 }
 
+//------------------------------------------------------------------------------
+// Only one instance of a Packet Replication Engine (PRE) is expected in the
+// P4 pipeline. Hence, no instance id is needed to access the PRE.
+message PreEntry {
+  oneof pre_entry {
+    MulticastGroupEntry multicast_group_entry = 1;
+    CloneSessionEntry clone_session_entry = 2;
+  }
+}
+
+// The egress instance must be unique for each replica in a given multicast
+// group entry.
+message MulticastGroupEntry {
+  bytes multicast_group = 1;
```

Comment:
I thought we agreed this would be a `int64`. It makes it much easier to manipulate it in software. We can ask the PSA group to constraint the bitwidth to <= 64-bit for the corresponding metadata field.

Reply:
Ya, I thought about it and put it on hold. We are not doing this for match fields or action parameters, so a multicast group as action parameter would be bytes, but as a PRE entry field would be int64. It seems a little inconsistent, but I agree it is a pain to manipulate bytes in SW and having to take care of endianness etc.

Reply:
This is not an arbitrary user-defined field though, this is an architecture-defined field. In a way we are hard-coding a name for it, so we can also hard-code a type for it.

Reply:
Fair point. In that case, we should make all fields uint64 (including ports).